main.go

```diff
@@ -265,6 +271,23 @@ func main() {
 	<-ctx.Done()
 }
 
+func getSriovTokenServerChainElement(ctx context.Context) (tokenServer networkservice.NetworkServiceServer) {
+	sriovTokens := tokens.FromEnv(os.Environ())
+	switch len(sriovTokens) {
+	case 0:
+		tokenServer = null.NewServer()
+	case 1:
+		var tokenKey string
+		for tokenKey = range sriovTokens {
+			break
+		}
```

Comment:
```suggestion
		var tokenKey = sriovTokens[0]
```

Reply:
It is a map, so it doesn't look like there is any other way to get a first key without creating some utility method.

Reply:
ah it is map, I've missed a type. OK thens its fine.